src/app/locator/qgsinbuiltlocatorfilters.cpp

```diff
@@ -1185,3 +1188,30 @@ void QgsGotoLocatorFilter::triggerResult( const QgsLocatorResult &result )
 
   mapCanvas->flashGeometries( QList< QgsGeometry >() << QgsGeometry::fromPointXY( point ) );
 }
+
+QgsNominatimLocatorFilter::QgsNominatimLocatorFilter( QgsGeocoderInterface *geocoder, QgsMapCanvas *canvas )
+  : QgsGeocoderLocatorFilter( QStringLiteral( "nominatimgeocoder" ), tr( "Nominatim Geocoder" ), QStringLiteral( "nom" ), geocoder, canvas )
```

Comment:
If we're shipping this as the out-of-the-box place searching method, could we pick a nicer prefix? Maybe a generic single character option like ":" or ">" or ...?

Reply:
Hmm maybe a question mark?

Reply:
I thought about that, but we probably should keep ? free for a future "search for help on ...." filter...

Reply:
OK, then I'd go for > , which can be seen as an arrow head, indicating we'll "go somewhere" :)